testing/tests/DevExpress.ui/collectionWidgetParts/editingTests.js

```diff
@@ -2128,6 +2162,27 @@ module('reordering of items', () => {
         instance.reorderItem(item(0), item(1));
     });
 
+    test('itemReordered event should be fired if items reordered', function(assert) {
+        const items = [{ a: 0 }, { a: 1 }];
+
+        const $element = $('#cmp');
+        const instance = new TestComponent($element, {
+            items: items
+        });
```

Comment:
`$element` and `items` are used once and can be inlined:
```suggestion
        const instance = new TestComponent('#cmp', {
            items: [{ a: 0 }, { a: 1 }]
        });
```